packages/swapper/src/swappers/zrx/utils/helpers/helpers.ts

```diff
@@ -44,8 +44,7 @@ type GrantAllowanceArgs = {
  * ETH or ERC20, we still keep 7 decimal places.
  * @param amount
  */
-export const normalizeAmount = (amount: string | undefined): string | undefined => {
-  if (!amount) return
+export const normalizeAmount = (amount: string): string => {
```

Comment:
```suggestion
export const normalizeAmount = (amount: string | number | BigNumber): string => {
```

don't we want this to be more flexible, it was the return type that was the problem? returning `string` is correct - the previous signature of `string | undefined` didn't make sense given the name of the function.

Reply:
gotcha. makes sense - I re added the input types